gen/tests/test_adminrouter_tls_conf.py

```diff
@@ -12,15 +12,18 @@ class TestAdminRouterTLSConfig:
     Tests for the Admin Router TLS Config creation.
     """
 
-    def test_default(self):
+    @pytest.mark.parametrize(
+        'adminrouter_tls_1_0_enabled, tls_versions',
+        [('true', 'TLSv1 TLSv1.1 TLSv1.2'), ('false', 'TLSv1.1 TLSv1.2')]
+    )
+    def test_master(self, adminrouter_tls_1_0_enabled, tls_versions):
         """
-        By default, the configuration specifies certain TLS settings.
-
-        This test is a sanity check for the configuration template logic
-        rather than a particularly useful feature test.
+        Test that Master Admin Router config file has the correct content.
```

Comment:
I think that the comment:

```
-        This test is a sanity check for the configuration template logic		
 -        rather than a particularly useful feature test.
```

is useful.

That is - what is "correct content"? How is "correct content" chosen and defined?

What we really mean is that we have tested this at a higher level, and this file happens to contain certain contents when the TLS settings work as promised. This is a regression test which will give us early indication that a larger test will fail, but that's it.

This comment is misleading.

Reply:
As @mhrabovcin is away, I will create a JIRA issue to address this and other non-critical comments.